modules/programs/autorandr.nix

```diff
@@ -264,6 +320,19 @@ in
   };
 
   config = mkIf cfg.enable {
+    assertions = flatten (mapAttrsToList (
+      profile: { config, ... }: mapAttrsToList (
+        output: opts: {
+          assertion = opts.scale == null || opts.transform == null;
+          message = ''
+            Cannot use the profile output options 'scale' and 'transform' simultaneously.
+            Check configuration for profile ${profile} and output ${output}.
```

Comment:
Instead of

> for profile ${profile} and output ${output}.

you could have

> for programs.autorandr.profiles.${profile}.config.${output}

which will directly show the option submodule that is problematic.